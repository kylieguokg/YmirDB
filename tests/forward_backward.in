set a 1
set f 1
set g 1
set b a
set c b
set e f g
set d c e
list entries
forward a
forward b
forward c
forward d
forward e
forward f
forward g
backward a
backward b
backward c
backward d
backward e
backward f
backward g
pop d
pop d
list entries
forward a
forward b
forward c
forward d
forward e
forward f
forward g
backward a
backward b
backward c
backward d
backward e
backward f
backward g
bye
