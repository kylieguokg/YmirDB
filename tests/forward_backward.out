> ok

> ok

> ok

> ok

> ok

> ok

> ok

> d [c e]
e [f g]
c [b]
b [a]
g [1]
f [1]
a [1]

> nil

> a

> a, b

> a, b, c, e, f, g

> f, g

> nil

> nil

> b, c, d

> c, d

> d

> nil

> d

> d, e

> d, e

> c

> e

> d []
e [f g]
c [b]
b [a]
g [1]
f [1]
a [1]

> nil

> a

> a, b

> nil

> f, g

> nil

> nil

> b, c

> c

> nil

> nil

> nil

> e

> e

> bye
